content/applications/sales/point_of_sale/payment_methods/terminals.rst

```diff
@@ -15,15 +15,14 @@ Configuration
 Go to the :ref:`application settings <configuration/settings>`, scroll down to the
 :guilabel:`Payment Terminals` section, and tick your terminal's checkbox.
 
-.. image:: terminals/settings-pt.png
+.. image:: terminals/payment-terminals.png
```

Comment:
Since this screenshot was taken outside of Odoo 15, I'd remove it.
In both Odoo 15 and Odoo 16, Vantiv still appears as an option...

Can add it back in on the 17.0 forward-port PR, if you want!